scipy/stats/tests/test_stats.py

```diff
@@ -4157,8 +4173,10 @@ def testLargeBoth(self):
     def testNamedAttributes(self):
         # test for namedtuple attribute results
         attributes = ('statistic', 'pvalue')
+        additional_attributes = ('statistic_location', 'statistic_sign')
```

Comment:
But why add this? It's shown that these attributes exist in both `test_location_sign` and `test_nametuples_agree`.

Reply:
You're right, of course. It's just that I have to watch my baby while coding, which makes these things hard to catch 😅

If you were as kind, it would be just fantastic if you could push this change to the PR branch and merge so we are fine with it. Otherwise, I'll try to find time next Wednesday to come back to this... Thank you!